commands/certmanager/post.go

```diff
@@ -0,0 +1,123 @@
+package certmanager
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/spf13/viper"
+)
+
+var createProperties = sdkgo.CertificatePostPropertiesDto{}
+
+func CertCreateCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace:  "certmanager",
+		Resource:   "certificates",
+		Verb:       "create",
+		Aliases:    []string{"c"},
+		ShortDesc:  "Add a new Certificate",
+		LongDesc:   "Use this command to add a Certificate.",
+		Example:    "ionosctl certificate-manager create",
+		PreCmdRun:  PreCmdPost,
+		CmdRun:     CmdPost,
+		InitClient: true,
+	})
+
+	cmd.AddStringFlag(FlagCertName, "n", "", "Specify name of the certificate (required either this or --certificate-name-path)")
+	cmd.AddStringFlag(FlagCert, "", "", "Specify the certificate itself (required either this or --certificate-path)")
+	cmd.AddStringFlag(FlagCertChain, "", "", "Specify the certificate chain (required either this or --certificate-chain-path)")
+	cmd.AddStringFlag(FlagPrivateKey, "", "", "Specify the private key (required either this or --private-key-path)")
+
+	cmd.AddStringFlag(FlagCertPath, "", "", "Specify the certificate itself from a file (required either this or --certificate)")
+	cmd.AddStringFlag(FlagCertChainPath, "", "", "Specify the certificate chain from a file (required either this or --certificate-chain)")
+	cmd.AddStringFlag(FlagPrivateKeyPath, "", "", "Specify the private key from a file (required either this or --private-key)")
+
+	return cmd
+}
+
+func GetPropertyWithFallback(c *core.CommandConfig, property string, propertyPath string) (string, error) {
+	propertyValue, err := c.Command.Command.Flags().GetString(property)
+	if err != nil {
+		return "", err
+	}
+	if propertyValue == "" {
```

Comment:
Here I would prefer early returns for readability, but its up to you:

```
if propertyValue != "" {
    return propertyValue, nil
}
// propertyValuePath handling below
```

https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement